crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -3448,39 +3449,34 @@
           {
             "type": "object",
             "required": [
-              "bcs_bytes",
               "dataType",
+              "fields",
               "has_public_transfer",
-              "type",
-              "version"
+              "type"
             ],
             "properties": {
-              "bcs_bytes": {
```

Comment:
supposedly this PR should not break the "schema", why do we have generated changes here?

Reply:
Nicely spotted lol, there was a schema clash between Raw and parsed data, the old schema wrongly picked up the “RawData” schema. This is due to the generic data structure, I will change them to enum in next PR